docs/pages/config.mdx

```diff
@@ -17,6 +17,14 @@ The is an example of a `World` config:
 ```tsx
 import { defineWorld } from "@latticexyz/world";
 
+// Used for the module parameters, not required otherwise.
+// If you use this, make sure to run:
+//
+// pnpm install @latticexyz/common
+//
+// In the packages/contracts directory.
+import { resourceToHex } from "@latticexyz/common";
```

Comment:
this is importing the wrong thing right? seems like you use `resolveTableId`, not `resourceToHex`

Reply:
Yes, I should have fixed it when I changed it below. Thank you for catching this - fixed.